tests/src/PatternAssertion/CarouselPatternAssert.php

```diff
@@ -98,20 +98,23 @@ protected function assertItems(array $expected, Crawler $crawler): void {
     foreach ($expected as $index => $expected_item) {
       $item = $items->eq($index);
 
+      // Check if HTML content contains video or iframe.
+      $isPlayable = strpos($item->html(), '<video') !== FALSE || strpos($item->html(), '<iframe') !== FALSE;
+
       try {
         self::assertStringContainsString($expected_item['image'], $item->html());
         $this->assertElementText($expected_item['caption_title'] ?? NULL, '.carousel-caption .fs-5', $item);
-
+        $this->assertElementAttribute($expected_item['interval'] ?? 0, '.carousel-item', 'data-bs-interval', $item);
```

Comment:
an $expected_item['interval'] is never set

Reply:
i made a dump for this on test 'carousel_all_item_values' and it's populated on my side.
I made another dump for one which is not set and it's undefined. So it works as expected.

Reply:
Its just that it was used in the carousel test, but not gallery, so it threw me off a bit, but its ok.